docs/faq.md

```diff
@@ -18,7 +18,69 @@ To manually run a bounded load-test using the [`coordinator`](https://github.com
 We define a load-test to be unbounded when the number of simulated users can vary up to a pre-configured limit.
 This type of load-test is used to determine the capacity of a system and will output an estimated number of users.
 The rule of thumb is that when starting an unbounded load-test we should always shoot for more users than what we think an installation can support.
-`ClusterConfig.MaxActiveUsers`  should be set to  `AgentInstanceCount * UsersConfiguration.MaxActiveUsers`.
+`ClusterConfig.MaxActiveUsers` should be set to `AgentInstanceCount * UsersConfiguration.MaxActiveUsers`.
+
+### Can I use a pre-existing Mattermost or database deployment?
+
+Yes, you can use an existing Mattermost deployment, or just the database portion.
+
+Note: You should **not** utilize an existing production setup to loadtest against because the loadtest agent will create users, posts, teams and channels to utilize and utilize most of your server resources. Best practice is to clone your production setup and loadtest against that.
```

Comment:
```suggestion
Note: You should **not** utilize an existing production setup to loadtest against because the loadtest agent will create users, posts, teams and channels and utilize most of your server resources. Best practice is to clone your production setup and loadtest against that.
```